network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ReactiveCapabilityCurveAdderImpl.java

```diff
@@ -45,6 +47,14 @@ public ReactiveCapabilityCurve add() {
         if (points.size() < 2) {
             throw new ValidationException(owner, "a reactive capability curve should have at least two points");
         }
+        Set<Map.Entry<Double, ReactiveCapabilityCurvePointAttributes>> entries = points.entrySet();
+        entries.forEach(point -> {
+            double minQ = point.getValue().getMinQ();
+            double maxQ = point.getValue().getMaxQ();
+            if (minQ > maxQ) {
+                throw new ValidationException(owner, "maximum reactive power " + maxQ + " is expected to be greater than or equal to minimum reactive power" + minQ);
+            }
+        });
```

Comment:
This check between minQ and maxQ must be implemented in PointAdderImpl.java endPoint() method

Reply:
done